### PR TITLE
[1.5 backport] pre-pull images before stopping docker

### PR DESCRIPTION
**Pull request by ingvagabund**
Backports https://github.com/openshift/openshift-ansible/pull/4666 to 3.5

### CODE REVIEW

**Comment by ingvagabund** [comment]
aos-ci-test

**Comment by tbielawa** [comment]
[test]

File: roles/openshift_node_upgrade/tasks/main.yml [hunk 1]
```diff
@@ -31,6 +31,20 @@
   failed_when: false
   when: openshift.common.is_containerized | bool
 
+- name: Pre-pull node image
+  command: >
```
Comment (kwoodson) [inline roles/openshift_node_upgrade/tasks/main.yml]: Can we leverage the ansible docker role?

http://docs.ansible.com/ansible/docker_module.html

**Comment by ashcrow** [comment]
No good information on why this failed:

```
RUNNING HANDLER [openshift_node : restart node] ********************************
fatal: [localhost]: FAILED! => {
    "changed": false, 
    "failed": true, 
    "generated_timestamp": "2017-07-06 09:45:51.239829", 
    "msg": "Unable to restart service origin-node: Job for origin-node.service failed because the control process exited with error code. See \"systemctl status origin-node.service\" and \"journalctl -xe\" for details.\n"
```

**Comment by sdodson** [comment]
Going to go ahead and merge this to bring the older branches in line with where master is.

**Pull request merged**